rpcserverhelp.go

```diff
@@ -536,10 +536,10 @@ var helpDescsEnUS = map[string]string{
 	"searchrawtransactions--result0":    "Hex-encoded serialized transaction",
 
 	// SendRawTransactionCmd help.
-	"sendrawtransaction--synopsis":     "Submits the serialized, hex-encoded transaction to the local peer and relays it to the network.",
-	"sendrawtransaction-hextx":         "Serialized, hex-encoded signed transaction",
-	"sendrawtransaction-allowhighfees": "Whether or not to allow insanely high fees (btcd does not yet implement this parameter, so it has no effect)",
-	"sendrawtransaction--result0":      "The hash of the transaction",
+	"sendrawtransaction--synopsis":  "Submits the serialized, hex-encoded transaction to the local peer and relays it to the network.",
+	"sendrawtransaction-hextx":      "Serialized, hex-encoded signed transaction",
+	"sendrawtransaction-maxfeerate": "Reject transactions whose fee rate is higher than the specified value. The value is given in satoshis",
```

Comment:
if the intent is to match the bitcoind implementation, the `maxfeerate` is in sats/kB and not satoshis

https://github.com/bitcoin/bitcoin/pull/13541/files#diff-01aa7d1d32f1b9e5a836c9c411978918R1043

Reply:
fixed